MAINTAINERS

```diff
@@ -0,0 +1,14 @@
+# Org-Level Maintainers (in alphabetical order)
+# Pattern: [First Name] [Last Name] <[Email Address]> ([GitHub Handle])
+Justin Cormack <justin.cormack@docker.com> (@justincormack)
+Niaz Khan <niazfk@amazon.com> (@niazfk)
+Steve Lasker <StevenLasker@hotmail.com> (@stevelasker)
+
+# Repo-Level Maintainers (in alphabetical order)
+# Pattern: [First Name] [Last Name] <[Email Address]> ([GitHub Handle])
```

Comment:
I think we need to add all subproject owners to this file. This list will be used as a central place to show all Notary maintainers and will be synced to http://maintainers.cncf.io/.

Reply:
The CODEOWNERS file is special in this way as it can work for all subprojects. The file does not automatically imply permission though. We still need to update the teams as per the information in the file

Reply:
@FeynmanZhou we need to figure out how to do that for the sub-projects. Adding more folks to this one will add them as approvals to the `.github` approval, which I think should be fine because they should also participate in the governance. We need to have the maintainers for sub-projects approved first though. Will create a new PR for those once we have those cleared.

Reply:
@FeynmanZhou Let's file another issue in this repo to synch the maintainers list with CNCF.